inference-engine/tests/functional/plugin/cpu/test_utils/cpu_test_utils.cpp

```diff
@@ -55,9 +55,13 @@ cpu_memory_format_t CPUTestsBase::cpu_str2fmt(const char *str) {
 }
 
 std::string CPUTestsBase::fmts2str(const std::vector<cpu_memory_format_t> &fmts) {
+    return fmts2strWithPrefix(fmts, "cpu:");
```

Comment:
The idea is to remove colon character from the test name, m.b. it is better to use yet another default prefix?

Reply:
this prefix is actually used is the source code as well (see MKLDDNNode constructor), so we cannot just change it.